doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1005,7 +1005,162 @@ Response code:
             "temperature"
         ]
 
-## Standard OMA-NGSI operations [/v2/queryContext]
+## Context Subscriptions [/v2/entities/{entityId}/subscriptions]
+
+This section is still under discussion.
+
+A subscription is represented by a JSON object with the following fields:
+
+* 'id': Subscription unique identifier
+* 'subject' : It s an object that describes the subject of the subscription.
+* 'notification' : It is an object that describes the notification received by the subscriber.
+* 'options' : It is a dictionary which can contain different parameters related to the subscription, namely the duration.
+
+A 'subject' contains the following subfields:
+
+* 'entity' : Id or pattern id which identifies the affected entities (optional)
+* 'type' : Type of the affected entities (optional)
+* 'attributes': Attributes to be monitored (if not specified it would mean any)
```

Comment:
(if not specified it would mean any) -> (if not specified it would mean all attributes)

Just a suggestion... I'm not sure what is better in English ;)